site/content/en/docs/tasks/manage/monitor_pending_workloads/pending_workloads_on_demand.md

```diff
@@ -260,7 +260,7 @@ You can pass optional query parameters:
 To view only 1 pending workloads use, starting from position 1 in LocalQueue run:
 
 ```shell
-kubectl get --raw "/apis/visibility.kueue.x-k8s.io/v1alpha1/localqueues/user-queue/pendingworkloads?limit=1&offset=1"
+kubectl get --raw "/apis/visibility.kueue.x-k8s.io/v1alpha1/namespaces/default/localqueues/user-queue/pendingworkloads?limit=1&offset=1"
```

Comment:
Did you check the new command on a live cluster (kind or cloud provider)? I remember going through the doc a while back and didn't encounter any issue here (or maybe forgot?).

Reply:
Yes, sure. I found that while I was checking :).

Reply:
Localqueue APIs should be executed with a namespace specified.

Reply:
Ah, yes it is correct here: https://kueue.sigs.k8s.io/docs/tasks/manage/monitor_pending_workloads/pending_workloads_on_demand/#local-queue-visibility, but not here.